NuKeeper/Configuration/CommandLineParser.cs

```diff
@@ -35,8 +35,18 @@ private static RepositoryModeSettings ReadSettingsForRepositoryMode(CommandLineA
             var githubToken = settings.GithubToken;
             var githubRepoUri = settings.GithubRepositoryUri;
 
+            Uri githubApiBase;
+            if (settings.GithubApiEndpoint != null)
+            {
+                githubApiBase = settings.GithubApiEndpoint;
```

Comment:
do we want to take the repository name as input? or do we still want to parse it out of a uri - it leaves them having to post an argument like this 

`dotnet run mode=repository github_token=<GitToken> github_repository_uri=https://github.com/NuKeeperDotNet/NuKeeper github_api_endpoint=https://api.github.com`

rather than 

`C:\Code\NuKeeper\NuKeeper>dotnet run mode=organisation github_token=<GitToken> github_api_endpoint=https://api.github.com/ github_organisation_name=NuKeeperDotNet github_repository_name=NuKeeper `

wdyt?